uint16 address
uint8 count
uint8 slave
---
uint16[] registers
